src/nethsec/mwan/__init__.py

```diff
@@ -182,11 +185,14 @@ def store_policy(e_uci: EUci, name: str, interfaces: list[dict]) -> list[str]:
         list of changed configuration
 
     Raises:
-        ValidationError: if name is not unique
+        ValidationError: if name is not unique or length name > 15
     """
     changed_config = []
     # generate policy name
     policy_config_name = utils.get_id(name.lower())
+    #  test length of policy name
+    if len(policy_config_name) > 15:
```

Comment:
This condition will be triggered even if entered name has 13 chars.
Move the check before line 192.

Also, note that that `get_id` function will add a `ns_` prefix to all records.
So, if the name must be maximum 15 length, the limit should be 12 because get_id function adds 3 more chars.